aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -60,15 +62,12 @@ module aptos_framework::block {
         new_epoch_interval: u64,
     ) acquires BlockResource {
         system_addresses::assert_aptos_framework(aptos_framework);
+        assert!(new_epoch_interval > 0, error::invalid_argument(EINVALID_EPOCH_INTERVAL));
+
         let block_metadata = borrow_global_mut<BlockResource>(@aptos_framework);
         block_metadata.epoch_interval = new_epoch_interval;
     }
 
-    /// Helper function to determine whether this module has been initialized.
-    fun is_initialized(): bool {
```

Comment:
Oh oh. You did it already widely.

Reply:
I think it's unnecessary as these functions cannot be called outside of genesis anyway. And genesis can only be done once.

Reply:
How do you know that? An error in the adapter could lead to calling genesis twice.

Reply:
Which would fail very early on because of duplicate resource creation.